test.sh

```diff
@@ -20,14 +28,15 @@ if [ "x$CURRENT_ARCH" == "x" ]; then
     exit 1
 fi
 
+killall secretshare-server
```

Comment:
I added this to deal with an infuriating series of test failures that were happening because an old `secretshare-server` process hadn't been killed. This can happen if test.sh is interrupted or dies unexpectedly.

Maybe test.sh should write the PID to a PID file and then we should check whether that process is still running instead of using killall?